lang/java/reef-bridge-client/src/main/java/org/apache/reef/bridge/client/LocalSubmissionFromCS.java

```diff
@@ -80,6 +87,8 @@ private LocalSubmissionFromCS(final AvroJobSubmissionParameters avroJobSubmissio
     Validate.isTrue(tcpBeginPort >= 0, "The tcp start port given is < 0.");
     Validate.isTrue(tcpRangeCount > 0, "The tcp range given is <= 0.");
     Validate.isTrue(tcpTryCount > 0, "The tcp retry count given is <= 0.");
+    Validate.isTrue(StringUtils.isNotEmpty(driverStdoutPath), "The stdout path is empty.");
+    Validate.isTrue(StringUtils.isNotEmpty(driverStderrPath), "The stderr path is empty.");
```

Comment:
This is not mandatory from the user's view. The `LocalSubmissionFromCS` object is created from parameters passed from C#, which is generated by "our code". If the user does not specify anything, we will simply use the default value of `<LOG_DIR>/driver.stdout`.